CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ cmake_minimum_required(VERSION 3.1)
 
 project(base64 LANGUAGES C VERSION 0.4.0)
 
+cmake_policy(SET CMP0082 NEW)
+cmake_policy(SET CMP0060 NEW)
+cmake_policy(SET CMP0065 NEW)
+cmake_policy(SET CMP0127 NEW)
```

Comment:
I think you need to do the following here:
```cmake
if(POLICY CMP0127)
```
Policy [`CMP0127`](https://cmake.org/cmake/help/latest/policy/CMP0127.html) is quire new.